apps/pie-microsite/src/content/pages/designers/getting-started/overview.md

```diff
@@ -0,0 +1,60 @@
+
+{% notification {
+type: "information",
+title: "New to Figma?",
+message: "If you’re new to Figma, check [these tutorials](https://www.figma.com/resources/learn-design/?fuid=1093444461414143879) that will help you learn how to use the tool in no time."
+} %}
+
+---
+
+## How is PIE structured?
+
+We have divided our foundations, components and assets into multiple libraries, so they can be easily found and referenced. All of them can be found on the [PIE Team]() section in Figma.
+
+You’ll need to enable these libraries if you want to use the PIE design system for your products. If you don’t know how to do that, please read our [‘How to enable libraries in Figma’]() guide.
+
+
+{% contentPageImage {
+src: "../../../../../assets/img/designers/getting-started/libraries-wheel.svg",
+mobileSrc: "../../../../../assets/img/designers/getting-started/libraries-wheel_narrow.svg",
+width: "512px",
+alt: "A map of how Libraries are structured in Figma.",
+caption: "A map of how Libraries are structured in Figma."
+} %}
+
+
+### Foundations libraries
+
+These libraries contain the building blocks for our design system. In these libraries you’ll find the styles created using colour, typography, elevation, radius and spacing (among others). These foundations should remain consistent across components, products and pillars.
+
+PIE designers own and maintain our foundation libraries, which are split into three different themes (PIE 2.0 Light, PIE 2.0 Dark and Skip Light). Make sure you’re working from the right library, and check with the PIE team if you’re unsure about which one you should use.
+
+### Core Components libraries
+
+Our core component libraries are made up of all the components that are used by multiple business areas and teams. These include components like buttons, labels, chips, input fields (among others). These components should remain consistent across products and pillars.
+
+PIE designers own and maintain our components libraries, which are split into three different themes (PIE 2.0 Light, PIE 2.0 Dark and Skip Light). Make sure you’re working from the right library, and check with the PIE team if you’re unsure about which one you should use.
+
+### Asset libraries
+
+These libraries contain assets which can be used in our products, such as icons and illustrations. They are owned and maintained by PIE designers in collaboration with the JET Creative Studio (Brand).
+
+### Business Area libraries
+
+Business Area libraries were created to provide more flexibility to our teams. Components which are unique to a specific Business Area (such as Customer, Restaurant…) and would not be relevant in other ones live within these libraries.
+
+Business Area designers own these libraries, and PIE designers only help with maintaining them and making sure they are consistent with our Core Components libraries. You can learn more about how we support these libraries in our Getting Support section.
```

Comment:
How about a link, even if it's just a placeholder for now?
```suggestion
Business Area designers own these libraries, and PIE designers only help with maintaining them and making sure they are consistent with our Core Components libraries. You can learn more about how we support these libraries in our [Getting Support]() section.
```

Reply:
Yes, if there's a link in the design then we absolutely need to include empty links. Sorry, I didn't spot this